core/server/src/main/java/alluxio/master/file/meta/InodeDirectory.java

```diff
@@ -210,7 +230,8 @@ public static InodeDirectory fromJournalEntry(InodeDirectoryEntry entry) {
             .setPinned(entry.getPinned())
             .setLastModificationTimeMs(entry.getLastModificationTimeMs())
             .setPermissionStatus(permissionStatus)
-            .setMountPoint(entry.getMountPoint());
+            .setMountPoint(entry.getMountPoint())
+            .setDirectChildrenLoaded(entry.getDirectChildrenLoaded());
```

Comment:
if entry is old, and does not have the field, what does `getDirectChildrenLoaded()` return? Does it throw an Exception?